Draft of tutorial for Google's Public CA (#1213)
cert-manager/website: content/docs/tutorials/google-public-ca/README.md

---
title: "Configuring Google's Public Certificate Authority"
linktitle: "Configuring Google's Public Certificate Authority"
---

> For years we used `cert-manager` to provision TLS certificates from ZeroSSL. Their ACME service is free, but we've really gotten what we paid for. Service outages were common, and more recently ZeroSSL added undocumented rate limiting for HTTP requests to their ACME API. This change put us in the same situation as this `cert-manager` user: <https://github.com/cert-manager/cert-manager/issues/5867>. We began looking for alternative certificate authorities that support the ACME protocol.

Reviewer: "We" isn't defined here, but we can rephrase it so that "we" isn't needed.

```suggestion
cert-manager can be used for issuing free certificates with ZeroSSL and Let's Encrypt. 

ZeroSSL's is great to get started but doesn't work well for large scales use-cases. More specifically, ZeroSSL has a rate limit for HTTP requests to their ACME API.

For companies that need to go beyond ZeroSSL's and Let's Encrypt's limitations, Google's Public Certificate Authority is a good fit.
```

> About a year ago, Google announced ACME protocol support for their new Public Certificate Authority. This is the alternative we're looking for. We've configured our system to use this and below I'll show how you can too. We'll follow part of Google's tutorial, but instead of using `certbot`, we'll configure a `ClusterIssuer` and associated `Secret` for `cert-manager`. <https://cloud.google.com/certificate-manager/docs/public-ca-tutorial>

Reviewer: ```suggestion
In this tutorial, we will follow part of [Google's Public CA tutorial][public-ca-tutorial]. Instead of using `certbot`, we will configure a `ClusterIssuer` and associated `Secret` for cert-manager. 

[public-ca-tutorial]: https://cloud.google.com/certificate-manager/docs/public-ca-tutorial
```
